lib/codebreaker/game.rb

```diff
@@ -0,0 +1,103 @@
+require 'yaml'
+
+class Game
+  TURNS = 10
+  HINT = 1
+
+  attr_reader :secret_code, :answer_user, :hint, :turns, :result, :message
+
+  def initialize
+    @secret_code = (1..4).map { rand(1..6) }
+    @answer_user = ''
+    @hint = HINT
+    @turns = TURNS
+    @result = []
+    @message = YAML.load(File.open(File.join(File.dirname(__FILE__), "../data/info.yml")))
```

Comment:
good, but I meant that we need to add some new class like `GameMessagesInterface` and add all logic to it (yaml load and all methods)
```
def initialize
  ...
  @game_message_interface = GameMessagesInterface.new
end
```